ava.config.js

```diff
@@ -3,8 +3,7 @@ export default {
 	files: ['**/*.test.[jt]s'],
 	require: ['ts-node/register'],
 	timeout: '1m',
-	faileFast: true,
-	extensions: ['ts', 'js'],
```

Comment:
Why remove the "extensions"? is that on purpose?

Reply:
Yakir told me we can remove it